server/game/basecard.js

```diff
@@ -247,6 +250,10 @@ class BaseCard {
         return !!this.traits[trait.toLowerCase()];
     }
 
+    isFaction(faction) {
```

Comment:
I think `hasFaction` would be a more consistent name with other methods we have (hasTrait, hasKeyword, etc).